src/content/structured/components/side-nav/code.mdx

```diff
@@ -747,6 +749,115 @@ export const withReactRouter = [
   </MemoryRouter>
 </ComponentPreview>
 
+### With MUI icons in SlottedSVG
```

Comment:
This might be a bit nit picky, but could you move this example above the React Router one? As other components tend to have the React Router variant as the last one, so if users went looking for it, it'd be good to be consistent